main.go

```diff
@@ -36,6 +40,9 @@ func main() {
 	status := flag.String("status", "all", "which availability_status to check")
 	flag.Parse()
 
+	// Check whether we need to skip empty sources when fetching them or not.
+	skipEmptySources := os.Getenv("SKIP_EMPTY_SOURCES") != "true"
```

Comment:
Kind of nit-picky...what do you think about wrapping the `os.Getenv()` call in a `strings.ToLower()`?  That way we don't get tripped up on "TRUE" vs "true" or "True" vs "true", etc

Reply:
It's actually a really good idea :smile: :+1: Thank you!